new/components/nginx/agent-files/celeryd-cloudify.conf.template

```diff
@@ -6,7 +6,7 @@ WORKER_MODIFIER="{{ worker_modifier }}"
 
 export BROKER_IP="{{ broker_ip }}"
 export MANAGEMENT_IP="{{ management_ip }}"
-export BROKER_URL="amqp://guest:guest@${BROKER_IP}:5672//"
+export BROKER_URL="amqp://{{ ctx.nodes.properties.rabbitmq_username }}:{{ ctx.nodes.properties.rabbitmq_password }}@${BROKER_IP}:5672//"
```

Comment:
this file is irrelevant and should be removed as it is a part of cloudify-agent itself.